indexer/indexers.py

```diff
@@ -40,9 +40,17 @@ class Indexer:
     """
 
     def __init__(self):
-        self.connection = connections.create_connection(
-            hosts=settings.ELASTICSEARCH['default']['hosts'], timeout=60
-        )
+        if settings.ELASTICSEARCH['default']['api_key']:
```

Comment:
```suggestion
        if settings.ELASTICSEARCH.get('default', []).get('api_key'):
```
Working around KeyErrors